package.json

```diff
@@ -52,7 +52,7 @@
     "ember-source": "~4.0.1",
     "ember-source-channel-url": "^3.0.0",
     "ember-template-lint": "^3.15.0",
-    "ember-try": "^2.0.0",
```

Comment:
this version fixes a copy issue that happens with 2.0.0 

for more [context](https://github.com/ember-cli/ember-try/issues/813)